tests/conftest.py

```diff
@@ -1772,6 +1773,12 @@ def label_registry(hass: HomeAssistant) -> lr.LabelRegistry:
     return lr.async_get(hass)
 
 
+@pytest.fixture
+def service_calls(hass: HomeAssistant) -> list[ServiceCall]:
+    """Track calls to a mock service."""
+    return async_mock_service(hass, "test", "automation")
```

Comment:
I think this is kinda misleading.
Instead of just mocking the `test.automation` service, we should mock all services here.

../Frenck

Reply:
I'm not sure what you mean about "all services".
Maybe the docstring should be adjusted to say 
`    """Track calls to test.automation service, can be overriden for other services."""`